src/components/SQFormDialogStepper/SQFormDialogStepper.tsx

```diff
@@ -113,14 +165,17 @@ export function SQFormDialogStepper({
     setCompleted([...completed, activeStep]);
   };
 
-  const handleStep = step => () => {
+  const handleStep = (step: number) => () => {
     const nextStep = step;
-    if ([nextStep].some(step => completed.includes(step))) {
+    if ([nextStep].some((step) => completed.includes(step))) {
```

Comment:
Im confused by this. wouldn't this just be the same as `if(completed.includes(step))`?

Reply:
I can look into it.  I actually did not modify this at all.  The parentheses were added to the function parameter to adhere to our current prettier config.